demo/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id "application"
+    id 'org.springframework.boot' version '2.4.4'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+}
+
+apply plugin: 'idea'
+apply plugin: 'java'
+
+group 'com.amplitude'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+}
+
+idea {
+    module {
+        sourceDirs += file('src')
+    }
+}
+
+sourceSets.main.java.srcDirs = ['src']
+
+dependencies {
+    implementation 'org.json:json:20201115'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+}
+
+application {
+    mainClassName = "com.amplitude.Demo"
+}
+
+task runWithJavaExec(type: JavaExec) {
+    group = "Execution"
+    description = "Run the main class with JavaExecTask"
+    classpath = sourceSets.main.runtimeClasspath
+    main = "com.amplitude"
```

Comment:
This is going to run the main method? Should this be `com.amplitude.Demo`? Also this is a micro service, why you bother running main?

Reply:
com.Amplitude.Demo was just for our internal testing of the API with `public static void main(String[] args)`

Ideally, yes, we would like a Gradle job like this one where the microservices app runs, sets up an endpoint that calls the Java SDK logEvent.